python/repositories.bzl

```diff
@@ -238,9 +239,20 @@ cc_library(
 
 exports_files(["python", "{python_path}"])
 
+# Used to only download coverage toolchain when the coverage is collected by
+# bazel.
+config_setting(
+    name = "coverage_enabled",
+    values = {{"collect_code_coverage": "true"}},
+)
+
```

Comment:
oh awesome, thank you for adding this!

Can you restrict visibility of this? It's not the py rule's responsibility to tell if coverage is enabled or not, so lets not expose a way for people to use it and think it is.